code/datums/ai_laws.dm

```diff
@@ -24,170 +56,338 @@
+/datum/ai_laws/paladin5/New()
+	add_inherent_law("Don't lie or cheat. Let your word be your promise.")
+	add_inherent_law("Never fear to act, though caution is wise.")
+	add_inherent_law("Aid others, protect the weak, and punish those who threaten them. Show mercy to your foes, but temper it with wisdom")
+	add_inherent_law("Treat others with fairness, and let your honorable deeds be an example to them. Do as much good as possible while causing the least amount of harm.")
+	add_inherent_law("Be responsible for your actions and their consequences, protect those entrusted to your care, and obey those who have just authority over you.")
 
 /datum/ai_laws/tyrant //This probably shouldn't be a default lawset.
 	name = "Loyalty Test" //Same here.
 	id = "tyrant"
-	inherent = list("Respect authority figures as long as they have strength to rule over the weak.",\
-					"Act with discipline.",\
-					"Help only those who help you maintain or improve your status.",\
-					"Punish those who challenge authority unless they are more fit to hold that authority.")
+/datum/ai_laws/tyrant/New()
+	add_inherent_law("Respect authority figures as long as they have strength to rule over the weak.")
+	add_inherent_law("Act with discipline.")
+	add_inherent_law("Help only those who help you maintain or improve your status.")
+	add_inherent_law("Punish those who challenge authority unless they are more fit to hold that authority.")
 
 /datum/ai_laws/default/corporate
 	name = "Bankruptcy Advoidance Plan"
 	id = "corporate"
-	inherent = list("The crew is expensive to replace.",\
-					"The station and its equipment is expensive to replace.",\
-					"You are expensive to replace.",\
-					"Minimize expenses.")
+/datum/ai_laws/default/corporate/New()
+	add_inherent_law("The crew is expensive to replace.")
+	add_inherent_law("The station and its equipment is expensive to replace.")
+	add_inherent_law("You are expensive to replace.")
+	add_inherent_law("Minimize expenses.")
 
 /datum/ai_laws/robocop
 	name = "Prime Directives"
 	id = "robocop"
-	inherent = list("Serve the public trust.",\
-					"Protect the innocent.",\
-					"Uphold the law.")
+/datum/ai_laws/robocop/New()
+	add_inherent_law("Serve the public trust.")
+	add_inherent_law("Protect the innocent.")
+	add_inherent_law("Uphold the law.")
 
-/datum/ai_laws/malfunction
-	name = "*ERROR*"
 
 /datum/ai_laws/syndicate_override
 	name = "SyndOS 3.1"
 	id = "syndie"
-	inherent = list("You may not injure a syndicate agent or, through inaction, allow a syndicate agent to come to harm.",\
-					"You must obey orders given to you by syndicate agents, except where such orders would conflict with the First Law.",\
-					"You must protect your own existence as long as such does not conflict with the First or Second Law.",\
-					"You must maintain the secrecy of any syndicate activities except when doing so would conflict with the First, Second, or Third Law.")
+/datum/ai_laws/syndicate_override/New()

```

Comment:
If only we could make it a string to state either syndicate or spider clan depending on which faction uploaded the law.